website/blog/2023-08-16-lets-make-identity-fun-again/item.md

```diff
@@ -28,7 +28,7 @@ The danger with letting identity become boring is that we’re not engaging in t
 
 > My pitch: Let’s make identity fun again. And in doing so, let’s think through a better way to decide whether to build or buy software.
 
-!["Image by <a href="https://pixabay.com/users/jplenio-7645255/?utm_source=link-attribution&utm_medium=referral&utm_campaign=image&utm_content=3092026">Joe</a> from <a href="https://pixabay.com//?utm_source=link-attribution&utm_medium=referral&utm_campaign=image&utm_content=3092026">Pixabay</a>"](./image1.jpg)
+![Image1](./image1.jpg "Image by jplenio on pixabay")
 
```

Comment:
It looks like Jen's intent here is to provide a link-through to the attribution.  Let's try this:
```
[![Image1](./image1.jpg "Image by jplenio on pixabay")](https://pixabay.com/users/jplenio-7645255/)
```

Reply:
"Image1" is meaningless here; that's the alt tag.  Consider providing a useful alt tag for vision-impaired users: "path through the woods by jplenio."  See _[Alt Text Best Practices](https://www.tiny.cloud/blog/alt-text-best-practices/#:~:text=You%20don't%20have%20to,could%20say%20%E2%80%9Cimage%20image%E2%80%9D.)_ for guidelines on alt tags and ADA compliance.

![image](https://github.com/goauthentik/authentik/assets/133134217/300f8c9a-2b7b-4bfd-9b87-f801107803b4)